.github/workflows/release.yml

```
@@ -110,77 +141,55 @@ jobs:
         pip wheel .
 
   create_release:
-    name: Create Release
-    runs-on: ubuntu-20.04
+    name: Create Release and Publish
+    runs-on: ubuntu-latest
```

I suggest that we stick with the lowest OS version we support, just to avoid having dependency issues.
```suggestion
    runs-on: ubuntu-20.04
```

---

In this case, I believe that the process we use to release a new release is not contradictory to the version we support :thinking:  and we would have to update this workflow every time we upgrade to a new version...

---

In the case of wheel, build, etc... ok!